### PR TITLE
NPC ability to open doors while moving now configuarble in npc_types

### PR DESCRIPTION
**Pull request by noudess**
This PR allows server operators to change the behavior of some mobs to not open doors.

The PR in itself changes no behavior, unless the operator changes the new field, can_open_doors to 0.  The npc_types is updated so that all npcs  _can_ open doors, which is the behavior before this PR.

My reason for wanting this is to quiet down home cities, where rodents are opening doors like there is no tomorrow, wearing out hinges, and my speakers.

### CODE REVIEW

File: zone/zonedb.cpp [hunk 1]
```diff
@@ -2728,6 +2729,7 @@ const NPCType *ZoneDatabase::LoadNPCTypesData(uint32 npc_type_id, bool bulk_load
 		temp_npctype_data->use_model        	= atoi(row[110]);
 		temp_npctype_data->flymode          	= atoi(row[111]);
 		temp_npctype_data->always_aggro	        = atoi(row[112]);
+		temp_npctype_data->can_open_doors		= atoi(row[113]);
```
Comment (KinglyKrab) [inline zone/zonedb.cpp:2732]: Should return a bool here just to be on the safe side.
```cpp
temp_npctype_data->can_open_doors		= atoi(row[113]) != 0 ? true : false;

**Review by KinglyKrab — APPROVED** [review]
Looks good to me, just had one small comment.

File: zone/zonedb.cpp (hunk 1 above)
Comment (mackal) [reply zone/zonedb.cpp:2732]: Just need the `!= 0` bit, it's already a bool with just that. (We do that in a lot of places)

Comment (noudess) [reply zone/zonedb.cpp]: Done.   Fixed my previous field as well.

Comment (KinglyKrab) [reply zone/zonedb.cpp]: 👍

**Comment by Akkadius** [comment]
Closing for preference of https://github.com/EQEmu/Server/pull/1231

**Pull request closed without merging**